drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -390,7 +394,8 @@ static void nrf5_tx_started(const struct device *dev,
 	}
 }
 
-#ifdef CONFIG_IEEE802154_NRF5_PKT_TXTIME
+/* This function cannot be used in the serialized version yet. */
+#if CONFIG_NET_PKT_TXTIME && !CONFIG_NRF_802154_SER_HOST
```

Comment:
with booelan CONFIG entries it should be `#if defined(CONFIG_...) && !defined(CONFIG_...)`

Reply:
Done.